poe-api/src/poe.py

```diff
@@ -508,6 +508,59 @@ def get_message_history(self, chatbot, count=25, cursor=None):
     messages = query_messages + messages
     return messages
 
+  def get_latest_messages(self, chatbot, count=25, cursor=None):
```

Comment:
Couldn't you just do `self.get_message_history(chatbot, count=count, cursor=cursor)[::-1])` instead of re-implementing the entire function?